tailwind-showcases/doc/tailwind-showcases.mdx

```diff
@@ -11,21 +10,29 @@ import theme from 'tailwindcss/defaultTheme.js';
 
 ```
 import { TailwindShowcases } from '@divriots/dockit-react';
-import theme from 'tailwindcss/defaultTheme.js';
+
+const partialTheme = {
+  extend: {
+    colors: {
+      primary: '#E43530',
+      ...
+    }
+  }
+};
 
 <TailwindShowcases
-  theme={theme}
+  theme={partialTheme}
   showcaseKey="opacity"
   componentProps={{ className: "shadow-xl h-32 w-32 bg-blue-700" }}
 />
 ```
 
 <TailwindShowcases
-  theme={theme}
+  theme={{}}
```

Comment:
maybe it could be optional ?

Reply:
This file is also useful for education perspective so it's better to have it explicitly.